actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Encode Content-Disposition filenames on `send_data` and `send_file`.
```

Comment:
This merits more detail. At minimum, we ought to name the specifications implemented (RFCs [2231](https://tools.ietf.org/html/rfc2231) and [5987](https://tools.ietf.org/html/rfc5987)) and provide examples.

Reply:
✅